docs/TRAINING.md

```diff
@@ -4,21 +4,17 @@
 
 You can use the `lumi train --model <model-type>` command to easily start training your model (with default config). Before anything else, try running it to see if everything is working. You'll need a dataset in tfrecord format in the default location (`./datasets/voc/tf`). See [DATASETS.md](./DATASETS.md) for more info.
 
-The `train` cli tool provides many options related to training.
+The `train` cli tool provides the following options related to training.
 
 Options:
   - `--config`/`-c`: Config file to use.
```

Comment:
I'd add a note indicating multiple configs may be used, and that they're read left-to-right (i.e. that the rightmost file has the most precedence).